larpandoracontent/LArMonitoring/VisualMonitoringAlgorithm.cc

```diff
@@ -150,6 +151,23 @@ void VisualMonitoringAlgorithm::VisualizeMCParticleList(const std::string &listN
         }
     }
 
+    for (const MCParticle *pMCParticle : *pMCParticleList)
+    {
+        const LArMCParticle *const pLArMCParticle(dynamic_cast<const LArMCParticle*>(pMCParticle));
+        const std::vector<pandora::CartesianVector> steps(pLArMCParticle->GetMCStepPositions());
```

Comment:
Suggest const reference. There's definitely a typedef for `std::vector<pandora::CartesianVector>` if you'd like to use that (`CartesianPointList`?), and I think this is all `using namespace pandora`.